src/app/saga.ts

```diff
@@ -204,6 +204,7 @@ export interface RemoteConfigValues {
   networkTimeoutSeconds: number
   dappFavoritesEnabled: boolean
   celoNews: CeloNewsConfig
+  useTwelveWordSeedPhrase: boolean
```

Comment:
nit: to follow similar other properties we have
```suggestion
  twelveWordSeedPhraseEnabled: boolean
```

Reply:
Done in b1a5a1d4d9a4e90b601b5797f8e6c32cd78b30c9!